### ISSUE TITLE
SQLite text data stored in integer columns displayed right aligned and comma separated

### ISSUE THREAD

**Issue by ertankucukoglu** [HeidiSQL/HeidiSQL#1893]
**Preconditions**

* HeidiSQL version: 12.6.0.6782
* Database type and version: SQLite3
* OS: Windows 11

**Describe the bug**

**To Reproduce**

Check here for some details: https://dba.stackexchange.com/questions/106364/text-string-stored-in-sqlite-integer-column

**Screenshots or Crash reports**

![image](https://github.com/HeidiSQL/HeidiSQL/assets/14805520/e4a7d640-f03f-4930-9190-42958ab311e9)
Above capture, FilePath is integer type in create statement. So, data is right aligned and comma separated.
But it is not actually an integer data and should be left aligned and no comma separation applied.

BTW, this is not something I do. I am provided with the database and need to struggle with it now.

Thank you for your consideration.

**Comment by ansgarbecker** [comment]
Yes, this is not supported in HeidiSQL. The data type is detected once, and then used for all values in a grid. I see no way to display different types of data in one column.

**Comment by ertankucukoglu** [comment]
Will it be possible to remove comma separation based on column value being an integer or not?
